webapp/src/main/java/org/opentestsystem/rdw/admin/model/AppSettings.java

```diff
@@ -1,9 +1,17 @@
 package org.opentestsystem.rdw.admin.model;
 
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+@Component
 public class AppSettings {
 
+    @Value("${app.analytics.trackingId}")
     private String analyticsTrackingId;
 
+    @Value("${app.reporting.webapp.url}")
```

Comment:
I think the property should be `app.reporting-webapp-url`. The only reason to have it `app.reporting.webapp.url` would be if there were other properties that needed to be set like `app.reporting.fubar.shoesize` and that isn't really the case here: this is the reportingWebappUrl, a single property.